ui/packages/app/vite.config.js

```diff
@@ -117,7 +117,9 @@ export default defineConfig(({ mode }) => {
 		],
 		test: {
 			environment: "happy-dom",
-			include: ["**/*.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}"]
+			include: ["**/*.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}"],
+			setupFiles: "./setUpTests.ts",
+			globals: true
```

Comment:
I set `globals: true` (following this [tutorial](https://testing-library.com/docs/svelte-testing-library/setup/)) because that was the only way `expect` would have `toBeVisible` defined.

I tried adding `import "@testing-library/jest-dom";` in StaticVideo.test.ts but then it complained about not being able to find the module.

The downside is that then the `ts:check` command complained about not being able to find the `expect` so I disabled type checking on test files. 

There may be a better way to handle all of this lol.